ja/advanced/best-practice-performance.md

```diff
@@ -55,7 +55,7 @@ Node.js 4.0+ または io.js 2.1.0+ を使用している場合、アプリケ
 
 開発環境では、静的ファイルを提供するために [res.sendFile()](/{{ page.lang }}/4x/api.html#res.sendFile) を使用できます。しかし、この関数を実稼働環境では使用しないでください。この関数は、ファイル要求ごとにファイル・システムから読み取る必要があり、著しい遅延を発生させ、アプリケーション全体のパフォーマンスに影響を与えるためです。効率をはるかに向上させることができる [sendfile](http://linux.die.net/man/2/sendfile) システム呼び出しで `res.sendFile()` は実装*されない*点に注意してください。
 
-代わりに、Express アプリケーションにファイルを提供するために最適化された [serve-static](https://www.npmjs.com/package/serve-static) ミドルウェア (または同等の関数) を使用してください。
+代わりに Express アプリケーションにファイルを提供するために最適化された [serve-static](https://www.npmjs.com/package/serve-static) ミドルウェア (または同等の関数) を使用してください。
```

Comment:
The original uses the Japanese comma in the right way. Removing it does not make a major downgrade either.
 Add a line note